app/templates/organizations/organization/core-data/index.hbs

```diff
@@ -82,7 +82,7 @@
                 <Item>
                   <:label>Alternatieve naam</:label>
                   <:content>
-                    {{@model.organization.alternativeName}}
+                    {{join-array @model.organization.alternativeName}}
```

Comment:
Alternatively we could add a getter to the `Organization` class which would do the formatting, and use that here instead, but a generic helper seemed better.

Reply:
Agreed, the helper seems the better solution. In retrospect, I should have already added this back when introducing the alternative name functionality /Entered on/ [2024-08-13 Tue 10:06]